src/workerd/server/workerd.capnp

```diff
@@ -386,6 +386,17 @@ struct Worker {
       unsafeEval @23 :Void;
       # A simple binding that enables access to the UnsafeEval API.
 
+      volatileCache :group {
```

Comment:
I wonder if we want to think of a better name for this. "Volatile cache" is technically correct but sounds scary. Kind of tempted to just call it "memcache" (especially given it may eventually be backed by memcache) but I suppose that's a bit misleading at the moment...

Reply:
Agree — what seems most important to get across is that it is an in-memory cache — this answers "why use this instead of Cache API"?

And moment you say `memcached` I immediately grok what this is. And I understand that can be distributed, vs. volatileCache or just "in memory cache" I think volatile memory, think about cache for particular isolate, etc.

Reply:
How about `MemoryCache` as an alternative name for now. Later, we can hook this up to `memcached`.